src/controllers/playback/video/index.js

```diff
@@ -1184,6 +1184,20 @@ export default function (view) {
         }
     }
 
+    function autoSelectSubtitleTrack() {
+        const player = currentPlayer;
```

Comment:
We should pass the player as a function parameter to be consistent with the other functions.